app/lib/task/backend.dart

```diff
@@ -328,26 +330,22 @@ class TaskBackend {
     String packageName, {
     bool updateDependants = false,
   }) async {
-    var lastVersionCreated = initialTimestamp;
-    await withRetryTransaction(_db, (tx) async {
-      final pkgKey = _db.emptyKey.append(Package, id: packageName);
+    final package = await packageBackend.lookupPackage(packageName);
```

Comment:
I think the idea with `trackPackage` is that it will be called as soon as we detect changes to a package... loading from cache will sort of defeat the purpose won't it?

I do wonder if we call this frequently? Or do we only call it once per publishing event. Checkout `_scanForPackageUpdates` it tracks which events we've seen, right?

Reply:
Note: we could also cross-reference this last published timestamp with the one on the `Package` entity, discarding the cache if they differ.

Reply:
> The cache, esp. this version list cache should be purged

"should be"

-----------

> I think we may call it more than once per publishing event,

I'd be curious to see how?